### ISSUE TITLE
[Enhancement]: Free Trial Welcome Message Celebrations

### ISSUE THREAD

**Issue by adrianduffell** [Automattic/wc-calypso-bridge#1063]
### Describe the solution you'd like

We received feedback the welcome message is not celebratory enough.

> Banner feels underwhelming. Something feels like it should be more celebratory. You are in your free trial. Everything is awesome! Let’s celebrate that! 

![image](https://user-images.githubusercontent.com/9312929/229784315-12eb4d5b-bc77-42ae-b4fe-29a6eea2e5d5.png)

**Comment by that-mike-bal** [comment]
@adrianduffell I think that in the short term we'll need to prioritize other things. I think we can work with @laurajohnsonwoo and @tckasa to update the welcome email they get immediately to cover the messaging/sentiment we want to achieve here.

**Comment by dlcallaway** [comment]
Don't we have a fun celebration screen already? Is that just when you upgrade to paid? Feels like we could reuse and tweak if so. @verofasulo @lauroraa

**Comment by that-mike-bal** [comment]
We have this one and I opened an issue to make sure we use it for both plans https://github.com/orgs/Automattic/projects/562/views/7?pane=issue&itemId=23850712

**Comment by warrendholmes** [comment]
@FlorianBru Could you look into this? I think it dropped off as moved fast to launch. To me, we're repeating a welcome in the info box and above the task list — should we have one message that welcomes the user and promotes action to the plans page?

**Comment by lauroraa** [comment]
> @FlorianBru Could you look into this? I think it dropped off as moved fast to launch. To me, we're repeating a welcome in the info box and above the task list — should we have one message that welcomes the user and promotes action to the plans page?

Let me know if/when you need copy for this, @FlorianBru!

**Comment by FlorianBru** [comment]
>Let me know if/when you need copy for this, @FlorianBru!
Absolutely, as soon as I start working on this task I will definitely ping you @lauroraa
